SSW.Rules.AzFuncs/Persistence/RulesDbContext.cs

```diff
@@ -6,8 +6,6 @@ namespace SSW.Rules.AzFuncs.Persistence;
 public class RulesDbContext : CosmosContext
 {
     public IRepository<Bookmark> Bookmarks { get; set; }
-    public IRepository<SecretContent> SecretContents { get; set; }
-    public IRepository<User> Users { get; set; }
```

Comment:
Are you sure we dont need this table?

Reply:
These tables were only used on scope of secret content functions, if we remove those functions we can safely delete them as there are no other references to these tables